### DISCUSSION TITLE
TOML - taplo support

### DISCUSSION THREAD

**Discussion by haraldh** [intellij-rust/intellij-rust#10813] [IntelliJ Rust Plugin Development]
Would be cool to have auto formatting on save via [`taplo fmt`](https://taplo.tamasfe.dev/) the same as with `rustfmt`. Also it has a language server :)

**Comment by Trustpad-web** [comment]
That would be a great addition to a standalone Rust IDE. Integrating taplo fmt for automatic formatting on save, as well as a robust language server to provide advanced features like intellisense, refactoring, and more, would make for a highly productive and user-friendly Rust development experience, similar to what is available in specialized IDEs for other languages.
